runtime/compiler/control/J9Options.cpp

```diff
@@ -2028,30 +2028,30 @@ J9::Options::fePreProcess(void * base)
    if (!TR::Compiler->target.cpu.isZ())
       self()->setOption(TR_DisableAOTBytesCompression);
 
-   // Check for option -XX:JITaaSClient and/or -XX:JITaaSServer
+   // Check for option -XX:+UseJITServer and/or -XX:+StartAsJITServer
```

Comment:
In theory one could have the options with "-" as well (meaning I want to disable a feature that was set previously). Could we take into account those as well?
You would first determine if client mode is enabled by comparing the index of "-XX:-UseJITServer" to the index "-XX:+UseJITServer". Then do the same for server mode. If both the client and server mode are set, then compare the index of "-XX:+UseJITServer" to the index of "-XX:+StartAsJITServer"